physical/raft/raft.go

```diff
@@ -1174,14 +1232,15 @@ func (b *RaftBackend) AddPeer(ctx context.Context, peerID, clusterAddr string) e
 		return errors.New("raft storage autopilot is not initialized")
 	}
 
-	b.logger.Trace("adding server to raft via autopilot", "id", peerID)
-	return b.autopilot.AddServer(&autopilot.Server{
+	server := &autopilot.Server{
```

Comment:
This change seems gratuitous.

Reply:
This came about because I needed to debug/log this struct before it was passed to `AddServer()`, and I forgot to put it back after I removed the debug logging. The pattern where you directly return the object as you create it makes debugging harder imo. So, hardly gratuitous, but I can put it back.